### PR TITLE
Support installation with rnpm

### PR DESCRIPTION
**Pull request by sirnacnud**
Added support for easy installation with rnpm.

Also updated README installation instructions.

### CODE REVIEW

**Comment by kylekurz** [comment]
+1 to this. I installed via rnpm and had to go edit MainActivity.java.

**Comment by jhen0409** [comment]
👍

**Comment by sibelius** [comment]
👍

File: package.json [hunk 1]
```diff
@@ -19,5 +19,10 @@
     "android"
   ],
   "author": "Jack Hsu <jack.hsu@gmail.com> (http://jaysoo.ca/)",
-  "license": "ISC"
+  "license": "ISC",
+  "rnpm": {
+    "android": {
+      "packageInstance": "new ExtraDimensionsPackage(this)"
```
Comment (sibelius) [inline package.json:25]: after this PR https://github.com/jaysoo/react-native-extra-dimensions-android/pull/10

this line should be only "new ExtraDimensionsPackage()"

**Comment by sirnacnud** [comment]
Since the "this" parameter is no longer required, this change shouldn't be needed to install with rnpm, as "new ExtraDimensionsPackage()" is added by default. Can someone confirm this? It may be a week or so before I can test it.

**Comment by sibelius** [comment]
that's correct @sirnacnud

**Comment by Sunhat** [comment]
If anyone is interested, I've forked this repo and implemented support for rnpm. I've also ensured it works the current version of React Native 0.37

 It looks as if @jaysoo isn't here to maintain this project

**Comment by Sunhat** [comment]
This is now in the newest version - Thanks for contributing!

**Pull request closed without merging**